active/CommandBar/new-command-bar-features.md

```diff
@@ -0,0 +1,286 @@
+ 
+![Image of a cascading menu](images/nested-menuflyout.PNG)
+In order to create this cascading/nested menu today, you can use a class called [MenuFlyoutSubItem](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyoutsubitem?view=winrt-19041) as an item in the menu. 
+This class represents a "parent" MenuFlyout item. It has an
+[Items](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyout.items?view=winrt-19041) property, where you can assign all of its child items.  Here's the above example with labels showing which item is which type:
+
+![Anatomy of a MenuFlyout](images/anatomy-diagram.PNG)
+
+You also have the option of using [RadioMenuFlyoutItem](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem?view=winui-2.5), which has a [GroupName](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem.groupname?view=winui-2.5#Microsoft_UI_Xaml_Controls_RadioMenuFlyoutItem_GroupName) property. Items with the same GroupName will be a part of the same selection model - only one of the items in a Group can be "selected" and display the selection indicator at a time:
+
+![Image of a menu flyout with radio items](images/radiomenuflyoutitems.png)
+
+The issue arises when you want to combine these two concepts. It would be useful to have RadioMenuFlyoutItems have their own child items that participate in the same Group as their parent. This would allow for cleaner UIs by decreasing the size of MenuFlyouts that have a long list of RadioMenuFlyoutItems. See an example below:
+
+![Nested radio menu flyout example](images/commandbar2.png)
+
+### Issue #3: CommandBarFlyout does not have an "always expanded" state.
+Currently, a CBF with secondary commands always shows a […] button that allows the user to collapse the CBF into just the top bar (primary commands). There's currently no way to have a CBF with secondary commands that is always expanded when invoked. 
+
+When the CBF is invoked via touch, only the primary commands are shown (with the option for the user to expand the list of secondary commands). The developer is currently not able to control how the CBF appears when invoked, and cannot control whether it can be collapsed or not. 
+
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+### Feature #1. Built-in styles to support adding SplitButton to CommandBar and CommandBarFlyout
+WinUI will provide built in styles for SplitButton that will allow them to conform with the look and feel of an AppBarButton. These styles will change properties such as the SplitButton's height/width, font size, color, interactions (color changes), and more. 
```

Comment:
Does the SplitButton look different when in the command bar's overflow?

Reply:
I imagine it would look like a menu item with sub menu items, but you can click on either the arrow or the main menu item.

Reply:
I imagined that it would take on the CommandBarFlyout styling if it's in an overflow menu. Is that feasible?

Reply:
Don't all the existing command bar controls have a different style in the overflow menu? That's what [ICommandBarElement2.IsInOverflow](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ICommandBarElement2.IsInOverflow) is for.

Reply:
@MikeHillberg Yes, I believe you're right. So I think it would be feasible to add a style for when items are in the overflow menu as well.